code/modules/mob/living/carbon/carbon.dm

```diff
@@ -507,3 +507,23 @@ var/const/GALOSHES_DONT_HELP = 8
 /mob/living/carbon/check_ear_prot()
 	if(head && (head.flags & HEADBANGPROTECT))
 		return 1
+
+/mob/living/carbon/proc/accident(var/obj/item/Item)

```

Comment:
Whoever told you that is stupid.
It's always been an acceptable part of our standards for a variable to be the capital form of the first letter of the last segment of the path.

eg:
```var/obj/item/I```
<B>i</B>tem
<B>I</B>